packages/core/src/extensions/HyperlinkToolbar/HyperlinkToolbarPlugin.ts

```diff
@@ -255,6 +255,14 @@ class HyperlinkToolbarView {
 
         this.hyperlinkToolbar.hide();
       },
+      getReferenceRect: () =>
+        this.hyperlinkMark
+          ? posToDOMRect(
+              this.editor.view,
+              this.hyperlinkMarkRange!.from,
+              this.hyperlinkMarkRange!.to
+            )
+          : undefined,
```

Comment:
when would this be undefined?

is this expected, or should it throw an error?

Reply:
do we still need `referenceRect` in dynamicParams?

Reply:
It's expected, returns `undefined` only when the element is hidden, so the bounding box for it doesn't actually exist and it doesn't make sense to try to access it.